test/controllers/callback_controller_test.rb

```diff
@@ -17,6 +17,7 @@ class CallbackControllerTest < ActionController::TestCase
     TEST_SESSION = "this.is.a.user.session"
 
     setup do
+      Rails.application.reload_routes!
```

Comment:
I wonder if we should do this globally for every test, i.e. put it in `test_helper`, to reduce the chance of a similar regression being introduced later.

Reply:
I like doing this in `test_helper` better than a base class, since our tests are extending multiple classes.

Reply:
I'll look into making this change!